### ISSUE TITLE
Question: Access subfolder where user has no access share by ACL

### ISSUE THREAD

**Issue by acarlsen** [hierynomus/smbj#559]
First of all great library. It works great. 

I however have a use case I am unsure is supported.

I have a hidden share which denies user access, however a sub folder allows access.

C:\
^--- hiddenshare (shared via SMBv2, NTFS ACL denies access for the user I'm using to logon)
____^----subfolder1 (NTFS ACL grants access for the user I'm using to logon)

How do I list files on sub folder when I can not access share? Windows client can do this.

**Comment by SamerAdra** [comment]
@acarlsen I am facing the same issue. Have you found any workaround such as a different Java library?

**Comment by acarlsen** [comment]
No, I haven't found a solution other than to change access right for hidden share.
